src/libs/Pusher/pusher.ts

```diff
@@ -90,8 +93,9 @@ function callSocketEventCallbacks(eventName: SocketEventName, data?: EventCallba
  * @returns resolves when Pusher has connected
  */
 function init(args: Args, params?: unknown): Promise<void> {
-    initPromise = new Promise((resolve) => {
+    return new Promise((resolve) => {
```

Comment:
```suggestion
    return new Promise<void>((resolve) => {
```

NAB but required so we can chain with `resolveInitPromise`